lib/ex_doc/formatter/html.ex

```diff
@@ -53,29 +66,46 @@ defmodule ExDoc.Formatter.HTML do
     %{config | main: main || @main}
   end
 
+  defp generate_build(extras, generated_files, output) do
+    extras = for {file, _, _} <- extras, do: "#{file}.html"
+
+    build_files =
+      MapSet.new(generated_files ++ extras)
```

Comment:
Theoretically there is no need for MapSet. I would also call `Enum.flat_map(&[&1, "\n"])` to ensure all entries have a new line (including the last one).